esp/esp/program/modules/base.py

```diff
@@ -292,22 +292,15 @@ def getTemplate(self):
         if self.module.inline_template:
             baseDir = 'program/modules/'+self.__class__.__name__.lower()+'/'
             base_template = baseDir + self.module.inline_template
-            per_program_template = baseDir+'per_program/'+str(self.program.id)+ \
-                '_'+ self.module.inline_template
 
             #   Iterate over a bunch of reasons to return a template;
             #   if none of them come up true, return None.
             try:
-                get_template(per_program_template)
+                get_template(base_template)
                 if self.useTemplate():
-                    return per_program_template
+                    return base_template
             except TemplateDoesNotExist:
-                try:
-                    get_template(base_template)
-                    if self.useTemplate():
-                        return base_template
-                except TemplateDoesNotExist:
-                    pass
+                pass
 
             return 'program/modules/%s/%s' % (self.__class__.__name__.lower(), self.module.inline_template)
```

Comment:
Isn't this the same as `base_template`?  Can't you just return that unconditionally now?